mbed_lstools/lstools_linux_generic.py

```diff
@@ -16,14 +16,16 @@
 """
 
 import re
+import subprocess
```

Comment:
Looks like `subprocess` got added here but isn't being used, can you remove it?

Reply:
Yes. Agreed.